variants/FEATHER_F405/variant.cpp

```diff
@@ -52,15 +52,15 @@ const PinName digitalPin[] = {
   PC_3,
   PC_2,
   PC_1,   // D13
-  PA_4,   // D14 - A0
+  PB_7,   // D14 SDA
```

Comment:
this is OK but...traditional arduino has A0 starting at D14, is there a reason you change it?

Reply:
In general D14 and D15 are SDA/SCL
But this is up to you, if you want to kept A0 at D14.